### DISCUSSION TITLE
2024/03/24/acm/codeforces/CodeforcesRound922/

### DISCUSSION THREAD

**Discussion by giscus** [Hukeqing/Hukeqing.github.io#15] [General(讨论贴)]
# 2024/03/24/acm/codeforces/CodeforcesRound922/

Codeforces Round 922 (Div. 2) 个人写题记录

https://blog.mauve.icu/2024/03/24/acm/codeforces/CodeforcesRound922/

**Comment by kernelbin** [comment]
好厉害！快快！教我算法！！！

**Reply by Hukeqing** [reply]
不哦，应该是你教我哦
